<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineCasingDialog</class>
 <widget class="QDialog" name="LineCasingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>588</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>588</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Line Casing</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>340</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>556</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Description</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="inspection_intervall_sbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="installation_date_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>70</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="installation_date_chbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>75</y>
       <width>26</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_casing_number_edit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>196</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>55</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Installation Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>95</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Inspection Interval</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Line Casing Number</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inspection_intervall_unit_cbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>155</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Inspection Interval Unit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>15</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Casing Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="casing_type_cbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>226</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <widget class="QComboBox" name="network_cbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>241</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_of_last_inspection_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>200</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="date_of_last_inspection_chbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>205</y>
       <width>26</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>185</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of Last Inspection</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="length_sbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>71</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>56</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Length [m]</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="outer_height_sbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>111</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>96</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Outer Height [m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>141</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Outer Width [mm]</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="outer_width_sbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>156</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QComboBox" name="jacket_pipe_diameter_cbox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>201</y>
       <width>196</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>186</y>
       <width>126</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Jacket Pipe Diameter</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>line_casing_number_edit</tabstop>
  <tabstop>installation_date_chbox</tabstop>
  <tabstop>installation_date_edit</tabstop>
  <tabstop>inspection_intervall_sbox</tabstop>
  <tabstop>inspection_intervall_unit_cbox</tabstop>
  <tabstop>date_of_last_inspection_chbox</tabstop>
  <tabstop>date_of_last_inspection_edit</tabstop>
  <tabstop>casing_type_cbox</tabstop>
  <tabstop>length_sbox</tabstop>
  <tabstop>outer_height_sbox</tabstop>
  <tabstop>outer_width_sbox</tabstop>
  <tabstop>jacket_pipe_diameter_cbox</tabstop>
  <tabstop>network_cbox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LineCasingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LineCasingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
